docs/pages/includes/acme.mdx

```diff
@@ -0,0 +1,21 @@
+Let's Encrypt verifies that you control the domain name of your Teleport deployment by communicating with the HTTPS server listening on port 443 of your Teleport Proxy Service.
+
+You can configure Teleport to complete the Let's Encrypt verification process—called the ACME protocol—by running the following `teleport configure` command, where `tele.example.com` is the domain name of your Teleport cluster and `user@example.com` is an email address used for notifications (you can use any domain):
```

Comment:
This reads a little bit awkward to me. My understanding is that `teleport configure` doesn't actually configure anything, but rather spits out some configuration that you can then paste into your teleport.yaml.

The way this reads, it sounds like `teleport configure` is actually modifying the file for you.